vcr/deploy/deploy.go

```diff
@@ -379,12 +381,11 @@ func createPackage(ctx context.Context, opts *Options, uploadResp api.UploadResp
 
 	fmt.Fprintf(io.Out, "%s Package created: package_id=%q\n", c.SuccessIcon(), createPkgResp.PackageID)
 
-	fmt.Fprintf(io.Out, "Start Watching build logs for package %q ...\n", createPkgResp.PackageID)
+	fmt.Fprintf(io.Out, "Waiting for build to start...\n")
```

Comment:
prefix this log line with a couple spaces so that it aligns with the other logs, ie:

"  Waiting for build to start...\n"